src/library/Hooks/useValidatorFilters/index.tsx

```diff
@@ -78,7 +78,7 @@ export const useValidatorFilters = () => {
         continue;
       }
       const totalNominations = stake[addressBatchIndex].total_nominations ?? 0;
-      if (totalNominations < maxNominatorRewardedPerValidator) {
+      if (maxNominatorRewardedPerValidator.isGreaterThan(totalNominations)) {
```

Comment:
Again, the numbers have been switched so this should be `isGreaterThanOrEqualTo`. But since these are both big numbers now, it can be how it was before with the same operator.